Modules/_json.c

```diff
@@ -1456,7 +1490,9 @@ encoder_listencode_obj(PyEncoderObject *s, _PyUnicodeWriter *writer,
 
 static int
 encoder_encode_key_value(PyEncoderObject *s, _PyUnicodeWriter *writer, bool *first,
-                         PyObject *key, PyObject *value, Py_ssize_t indent_level)
+                         PyObject *key, PyObject *value, Py_ssize_t indent_level,
+                         PyObject *current_newline_indent,
+                         PyObject *current_item_separator)
```

Comment:
`current_newline_indent` and `current_item_separator` are pretty long names. What if remove the `current_` prefix from them?

Reply:
Removing the prefix works for this method, but in the other methods that take the same argument there already was a `newline_indent`. I took nineteendo's suggestion to rename those to `new_newline_indent`